backend/app.py

```diff
@@ -34,25 +36,55 @@ def get_surveys():
     data = db.executeSelectQuery(query)
     response = {}
     for d in data:
-        title, name, desc, s_id, q_id, q_text, qnumber, c_val, cnumber = d
-        pass
-
-    response = {
-        "Surveys" : [
-            {
-            "Id": None,
-            "Title": None,
-            "Name": None,
-            "Description": None,
-            "Question" : [{
-                "QNumber": None,
-                "Text": None,
-                "Choices" : [{
-                    "CNumber": None,
-                    "Value": None,
-                }]
-            }]
+        title, name, desc, s_id, qnumber, q_text, c_val, cnumber = d
+        if s_id not in response.keys():
+            response[s_id] = {
+                "Title": title,
+                "Name": name,
+                "Description": desc,
+                "Questions": {}
+            }
+        if qnumber not in response[s_id]["Questions"].keys():
+            response[s_id]["Questions"][qnumber] = {
+                "Text": q_text,
+                "Choices": {}
             }
-        ]
-    }
-    return response
+        if cnumber not in response[s_id]["Questions"][qnumber]["Choices"].keys():
+            response[s_id]["Questions"][qnumber]["Choices"][cnumber] = {
+                "Value": c_val
+            }
+
+    return jsonify(response)
+
+@app.route("/api/submit/survey", methods=['POST'])
```

Comment:
nit: again here you don't have to do the `/submit` portion in the route. same as above.